path_config_test.go

```diff
@@ -228,6 +228,68 @@ func TestConfig_Update(t *testing.T) {
 			t.Fatalf("no response expected, got: %#v", resp)
 		}
 	})
+
+	t.Run("update retains prior config values in storage", func(t *testing.T) {
```

Comment:
Should we add a test to document the expected behaviour for create operations on an existing config?

Reply:
Similar to above, it's actually not possible to hit this scenario since we don't define an `ExistenceCheck` here. The backend will always receive a `logical.UpdateOperation` (for both HTTP POST and PUT). Good idea if they were treated as different operations though 👍